packages/yoroi-extension/app/components/wallet/staking/dashboard-revamp/OverviewDialog.js

```diff
@@ -33,8 +33,12 @@ const messages = defineMessages({
   availableTotalRewardsHelper: {
     id: 'wallet.staking.availableTotalRewardsHelper',
     defaultMessage:
-      '!!!If the Reward amount is different than expected, see possible reasons of that on our FAQ page.',
+      '!!!If the Reward amount is different than expected, see possible reasons of that on our',
```

Comment:
Please change the string to:
```
'!!!If the Reward amount is different than expected, see possible reasons of that on our {faqLink}'
```
and handle it the same way `app/components/settings/categories/SupportSettings.js` handles `faqContent`.